### PR TITLE
Feature/159/fix donate

### PR DESCRIPTION
**Pull request by khanh-do**
### Impacted areas in the application: Donate Page

List general components of the application that this PR will affect: 
* styling

PR checklist
- [x] I included  a screenshot for FE changes
- [x] I have linked the PR to a Zenhub ticket
- [x] I have checked for merge conflicts
- [x] I have run the [prettier](https://prettier.io/) command `make pretty`

Screenshot from 2019-08-12 20-46-02

### CODE REVIEW

File: frontend/src/components/donation_page/index.jsx [hunk 1]
```diff
@@ -1,39 +1,39 @@
 import React, { Component } from "react";
+import { Container, Row, Col } from "react-bootstrap";
```
Comment (dkaplan88) [inline frontend/src/components/donation_page/index.jsx]: `Col` is not being used -> so lets remove that from here

File: frontend/src/components/donation_page/index.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.content-styling {
+  font-size: 32px;
```
Comment (dkaplan88) [inline frontend/src/components/donation_page/index.css]: this looks a little big to me? Maybe we go with 24px?

File: frontend/src/components/donation_page/index.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  margin-left: 266px;
+  margin-right: 266px;
```
Comment (dkaplan88) [inline frontend/src/components/donation_page/index.css]: can we add some styling for mobile? I do not think we want these margins at a lower a screen size.

File: frontend/src/components/donation_page/index.jsx (hunk 1 above)
Comment (khanh-do) [reply frontend/src/components/donation_page/index.jsx]: Done

File: frontend/src/components/donation_page/index.css (hunk 2 above)
Comment (khanh-do) [reply frontend/src/components/donation_page/index.css]: Done

File: frontend/src/components/donation_page/index.css (hunk 3 above)
Comment (khanh-do) [reply frontend/src/components/donation_page/index.css]: Done. I left the 266px for the large screen size as per the mock up directions.

**Pull request merged**